README.md

```diff
@@ -124,43 +126,50 @@ Change `x` to the number of your pull request.
 
 ## Run e2e-tests
 
-E2e-tests can be run against public cbioportal instances or against a local dockerized backend. These two e2e-tests types are referred to as `remote` and `local` types of e2e-tests.
+End-to-end tests can be run against public cbioportal instances or against a local dockerized backend. These two e2e-tests types are referred to as `remote` and `local` types of e2e-tests.
 
 ## Run of `remote e2e-tests`
 
-Install webdriver-manager, which manages standalone Selenium installation:
+First, install webdriver-manager, which manages standalone Selenium installation:
 ```
-yarn install -g webdriver-manager
+yarn global add webdriver-manager
```

Comment:
There is also a way to run webdriver-manager w/o installing globally which might be cleaner b/c of potential global version conflicts. One can execute the binary in node_modules/ directly. We do that in `.circleci/config.yml`:

https://github.com/cBioPortal/cbioportal-frontend/blob/master/.circleci/config.yml#L25

To avoid having to put that whole path of node_modules/ etc. One can also add a command to package.json and use that. Like a new script called `start-webdriver-manager` or something